src/gridstack.ts

```diff
@@ -164,7 +163,7 @@ export class GridStack {
     // create the grid element, but check if the passed 'parent' already has grid styling and should be used instead
     let el = parent;
     if (!parent.classList.contains('grid-stack')) {
-      let doc = document.implementation.createHTMLDocument();
+      let doc = document.implementation.createHTMLDocument("");

```

Comment:
use '' (2 single quote) instead, also you didn't answer me if IE require a param to be passed (if so add comment)

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation/createHTMLDocument
title  Optional (except in IE)
I see that now....